src/comp/comp.py

```diff
@@ -24,48 +24,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if "D" in human.name[:1]]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if "e" in human.name[len(human.name)-1]]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
+lettersCG = ["C", "D", "E", "F", "G"]
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans for letter in lettersCG if letter in human.name[:1]]
```

Comment:
This is a little tough to quickly understand in this form, but definitely works!